subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/BasicGroupedTaskLoggingFunctionalSpec.groovy

```diff
@@ -206,6 +206,37 @@ class BasicGroupedTaskLoggingFunctionalSpec extends AbstractConsoleFunctionalSpe
         gradle?.waitForFinish()
     }
 
+    def "group header is printed red if task failed"() {
+        given:
+        buildFile << """
+            task failing { doFirst { 
+                logger.quiet 'hello'
+                throw new RuntimeException('Failure...')
+            } }
+        """
+
+        when:
+        fails('failing')
+
+        then:
+        result.output.contains("[31;1m> Task :failing")
```

Comment:
Fixed. I didn't see `AbstractConsoleFunctionalSpec.styled()` before. Thanks for the pointers.